README.md

```diff
@@ -1,11 +1,11 @@
 Run amps with this config
 
 ```
-ampsServer --config ./config.xml
+ampServer --config ./config.xml
```

Comment:
should be `ampsServer` as the service is called `amps`

Reply:
Hmm, in the eval copy I got, the binary was `ampServer`.

```
davis@localhost:~$ cd AMPS-5.3.0.339-Release-Linux/
davis@localhost:~/AMPS-5.3.0.339-Release-Linux$ cd bin
davis@localhost:~/AMPS-5.3.0.339-Release-Linux/bin$ ls
amps_bio_perf_test     amps_fast_journal_dump   amps_journal_dump     amps_sow_dump        install-amps-daemon.sh
amps_clients_ack_dump  amps_file                amps_journal_search   amps-sqlite3         lib
ampserr                amps-grep                amps_queues_ack_dump  amps-sqlite3-report  sow
ampServer              amps-init-script-debian  amps-report-minidump  amps_upgrade         spark
ampServer-compat       amps-init-script-rhat    amps.service          config.xml           uninstall-amps-daemon.sh
```

Reply:
@texodus should this be changed? or were you being clever 😆

Reply:
The name of the bin is `ampServer`, with no "s".  We're not that clever.